### PR TITLE
Utility to get CTP rates

### PR DESCRIPTION
**Pull request by mpuccio**

### CODE REVIEW

**Comment by lietava** [comment]
HI @mpuccio , I suggested modification for adding also rate from ZNC class added after van der meer.

**Review by lietava — CHANGES_REQUESTED** [review]
This may work

File: Common/CCDB/ctpRateFetcher.cxx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+namespace o2
+{
+
+using framework::Service;
+
+double ctpRateFetcher::fetch(Service<o2::ccdb::BasicCCDBManager>& ccdb, uint64_t timeStamp, int runNumber, std::string sourceName)
+{
+  if (sourceName.find("ZNC") != std::string::npos) {
+    return fetchCTPratesInputs(ccdb, timeStamp, runNumber, 26) / (sourceName.find("hadronic") != std::string::npos ? 28. : 1.);
```
[the remark this answers is not in the document]
Comment (lietava) [reply Common/CCDB/ctpRateFetcher.cxx:31]: The improved C1ZNC is available from run 544448. So in wraper for ZNC we can:
if run < 544448 return input
else return class

File: Common/CCDB/ctpRateFetcher.cxx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    if (ctpcls[i].name == className) {
+      classIndex = i;
+      break;
+    }
+  }
+  if (classIndex == -1) {
+    LOG(fatal) << "Trigger class " << className << " not found in CTPConfiguration";
+  }
+
+  auto rate{mScalers->getRateGivenT(timeStamp, classIndex, 1)};
```
[the remark this answers is not in the document]
Comment (lietava) [reply Common/CCDB/ctpRateFetcher.cxx:69]: this is the code

**Comment by mpuccio** [comment]
Hi @ddobrigk, it would be great if we could merge this. We will also add a task to create a table with these information in the future, but having the utility for Pb-Pb where the rate changes often is quite useful.

**Review by ddobrigk — APPROVED** [review]

**Comment by ddobrigk** [comment]
> Hi @ddobrigk, it would be great if we could merge this when the CI is green. We will also add a task to create a table with these information in the future, but having the utility for Pb-Pb where the rate changes often is quite useful.

@mpuccio, sure: Approved and auto-merge enabled 👍

**Comment by mpuccio** [comment]
Thanks a lot!

**Pull request merged**